pep-0484.txt

```diff
@@ -861,6 +861,73 @@ allow all operations on it, and a value of type ``Any`` can be assigned
 to a variable (or used as a return value) of a more constrained type.
 
 
+Meta-types
+----------
```

Comment:
I think "Meta-types" is not a good name for this section for four reasons:

1. This could add to the classes vs. types confusion: Unlike metaclasses that are quite different from classes, types do not have such hierarchy, since one should not type annotate types, one can only type annotate classes.

2. ``Type[C]`` has normal semantics of type: This is just a set of objects that respond ``True`` to ``issubclass(objc, C)``.

3. I think someone who is looking for a documentation of this feature will not look for "Meta-types", therefore I would propose to change this to a simpler title, like "Type annotating class objects".

4. Finally, using such terminology could lead to proliferation of "meta": If we call annotations of values that are class objects by ``Type[C]`` or by ``ABCMeta`` meta-types, then how are we supposed to call annotations of values that are metaclasses (by ``Type[ABCMeta]`` for example)? Meta-meta-types? :-)